infrastructure/code/code_html_test.go

```diff
@@ -52,7 +52,7 @@ func Test_Code_Html_getCodeDetailsHtml(t *testing.T) {
 	}
 
 	// invoke method under test
-	codePanelHtml := getCodeDetailsHtml(issue)
+	codePanelHtml := getCodeDetailsHtml(issue, "")
```

Comment:
add a folderPath here and check the generated response

Reply:
Fixed!